src/NUnitFramework/framework/Internal/Builders/DefaultSuiteBuilder.cs

```diff
@@ -176,7 +176,7 @@ private bool HasArguments(IFixtureBuilder attr)
             // Only TestFixtureAttribute can be used without arguments
             var temp = attr as TestFixtureAttribute;
 
-            return temp == null || temp.Arguments.Length > 0 || temp.TypeArgs.Length > 0;
+            return temp == null || (temp.Arguments != null && temp.Arguments.Length > 0) || (temp.TypeArgs != null && temp.TypeArgs.Length > 0);
         }
```

Comment:
Actually if I write `[TestFixture(null)]` I'm probably intending `null` to be the first argument.
Another overload might be good, so that I don't have to write `[TestFixture(new object[] { null })]` which would be weird next to non-null usages of the attribute.